presto-spi/src/main/java/com/facebook/presto/spi/function/FunctionNamespaceManager.java

```diff
@@ -74,7 +74,7 @@
      * List all functions managed by the {@link FunctionNamespaceManager}.
      * TODO: Support transaction
      */
-    Collection<F> listFunctions();
+    Collection<F> listFunctions(Optional<String> likePattern);
```

Comment:
maybe call it `functionFilterPattern`? When I see `like` with `listFunctions`, I was under the impression that we gonna do something with `like()` function